x-pack/plugins/alerting/server/lib/get_execution_log_aggregation.ts

```diff
@@ -418,6 +420,14 @@ export function getExecutionLogAggregation({
                     },
                   },
                 },
+                maintenanceWindowIds: {
+                  top_hits: {
+                    size: 1,
+                    _source: {
+                      includes: MAINTENANCE_WINDOW_IDS_FIELD,
```

Comment:
can you just add `MAINTENANCE_WINDOW_IDS_FIELD` to the other `top_hits` aggregation?